utilities/rocAL/rocAL_performance_tests_with_depth/CMakeLists.txt

```diff
@@ -1,64 +0,0 @@
-################################################################################
-#
-# MIT License
-#
-# Copyright (c) 2018 - 2022 Advanced Micro Devices, Inc.
-#
-# Permission is hereby granted, free of charge, to any person obtaining a copy
-# of this software and associated documentation files (the "Software"), to deal
-# in the Software without restriction, including without limitation the rights
-# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-# copies of the Software, and to permit persons to whom the Software is
-# furnished to do so, subject to the following conditions:
-#
-# The above copyright notice and this permission notice shall be included in all
-# copies or substantial portions of the Software.
-#
-# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-# SOFTWARE.
-#
-################################################################################
-
-cmake_minimum_required (VERSION 3.0)
-
-project (rocAL_performance_tests_with_depth)
-
-set(CMAKE_CXX_STANDARD 11)
-set(ROCM_PATH /opt/rocm CACHE PATH "Default ROCm installation path")
-
-list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../amd_openvx/cmake)
-
-find_package(OpenCV QUIET)
-find_package(AMDRPP QUIET)
-
-include_directories(${ROCM_PATH}/libexec/mivisionx/rali/python/)
-link_directories(${ROCM_PATH}/lib/)
-file(GLOB My_Source_Files ./*.cpp)
-add_executable(${PROJECT_NAME} ${My_Source_Files})
-
-if(OpenCV_FOUND)
-    if(${OpenCV_VERSION_MAJOR} EQUAL 3 OR ${OpenCV_VERSION_MAJOR} EQUAL 4)
-        message("-- OpenCV Found -- Version-${OpenCV_VERSION_MAJOR}.${OpenCV_VERSION_MINOR}.X Supported")
-        include_directories(${OpenCV_INCLUDE_DIRS})
-        target_link_libraries(${PROJECT_NAME} ${OpenCV_LIBRARIES})
-        if(${OpenCV_VERSION_MAJOR} EQUAL 4)
-	        target_compile_definitions(${PROJECT_NAME} PUBLIC USE_OPENCV_4=1)
-        else()
-	        target_compile_definitions(${PROJECT_NAME} PUBLIC USE_OPENCV_4=0)
-        endif()
-    else()
-        message(FATAL_ERROR "OpenCV Found -- Version-${OpenCV_VERSION_MAJOR}.${OpenCV_VERSION_MINOR}.X Not Supported")
-    endif()
-else()
-    message(FATAL_ERROR "OpenCV Not Found -- No Display Support")
-endif()
-
-target_link_libraries(${PROJECT_NAME} rocal)
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3 -mf16c -Wall ")
-
-install(TARGETS ${PROJECT_NAME} DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
This CMakeLists.txt has been deleted Also after adding the file please add a newline at EOF